view/src/test/java/cmm/apps/esmorga/view/screenshot/eventdetails/EventDetailsScreenshotTest.kt

```diff
@@ -40,8 +47,11 @@ class EventDetailsScreenshotTest : BaseScreenshotTest() {
                         image = "test.png",
                         locationName = "Mi casa",
                         locationLat = lat,
-                        locationLng = lng
+                        locationLng = lng,
+                        primaryButtonTitle = getPrimaryButtonTitle(isAuthenticate, userJoined),
```

Comment:
You should not test this logic in a screenshot test. Screenshot tests are meant for checking the UI, business logic is tested in VM test and others. 

Even worse, since you are setting loading to true, this is never shown. Either creating 2 different test or set loading to false if you think loading is not that important to test.